src/common/translation/i18n/sv.json

```
@@ -557,16 +551,16 @@
         }
       },
       "child": {
-        "birthdate": {
+        "birthyear": {
           "input": {
-            "label": "Barnets födelsedatum (dd.mm.åååå)",
-            "placeholder": "Ange barnets födelsedatum"
+            "label": "Barnets födelsedatum (åååå)",
```

```suggestion
            "label": "Barnets födelseår",
```
i.e. "Child's birthyear" in Swedish